tests/acceptance/mock_server.py

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/python
+# Copyright 2020 Northern.tech AS
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+
+import os
+import pytest
+import tempfile
+
+from utils.common import put_no_sftp
+
+EXPIRATION_TIME = 10
+BOOT_EXPIRATION_TIME = 5
+
+MENDER_CONF = """{
+    "UpdatePollIntervalSeconds": 1,
+    "InventoryPollIntervalSeconds": 5,
+    "RetryPollIntervalSeconds": 1,
+    "UpdateControlMapExpirationTimeSeconds": %d,
+    "UpdateControlMapBootExpirationTimeSeconds": %d,
+    "ServerURL": "https://docker.mender.io:8443",
+    "DBus": {
+        "Enabled": true
+    }
+}
+""" % (
+    EXPIRATION_TIME,
+    BOOT_EXPIRATION_TIME,
+)
+
+
+@pytest.fixture
```

Comment:
Better to be explicit, I think
```suggestion
@pytest.fixture(scope="function")
```